drivers/net/bnxt/bnxt_ethdev.c

```diff
@@ -819,6 +820,78 @@ static int bnxt_rss_hash_update_op(struct rte_eth_dev *eth_dev,
 	return 0;
 }
 
+int rte_pmd_bnxt_set_tx_loopback(uint8_t port, uint8_t on)
```

Comment:
This method invoked by VFd, not supported by HWRM now, not sure whether we should keep the stub.